python/lsst/obs/pfs/__init__.py

```diff
@@ -1 +1,2 @@
+from .dm29117 import *  # Monkey-patch lsst.daf.base.PropertyList to fix DM-29117
 from .pfsMapper import *
```

Comment:
Is there a good reason for having wildcard imports, or can the imports be done explicitly?

Reply:
In `__init__.py`, you want to import everything that's being exported by the module (declared in `__all__`).